src/helpers/period.coffee

```diff
@@ -0,0 +1,7 @@
+_ = require 'underscore'
+
+PeriodHelper =
+  getOrder: (period) ->
+    parseInt(period.name)
```

Comment:
I belive @philschatz is correct. parseInt("Period 10") would return `NaN`.  

On a somewhat related note, you should always use the radix argument for parseInt, otherwise Javascript will choose base 8 or 16 if the first argument starts with a zero.  Probably unlikely in this case, but it's good practice to always use 10 for the radix.

Here I think you'll have to use replace with a regex: 
`parseInt(period.name.replace(/\D/g,''), 10)`